source/manual/troubleshoot-development-vm.html.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Troubleshoot your development VM
+parent: "/manual.html"
+layout: manual_layout
+section: Support
+owner_slack: "#govuk-developers"
+last_reviewed_on: 2017-08-08
+review_in: 3 months
+---
+
+Here are some things that can go wrong with the development VM.
+
+## Running out of disk space
+
+You may run out of disk space when replicating data into your development
+environment.
+
```

Comment:
I remember adding some attempt at logging when Postgres runs out of memory with you a while back, what do you reckon about mentioning that here?

We log to `/tmp/sync-postgresql-${PROD_DB_NAME}`